packages/sdk-vue/src/composables/useFusionAuth.ts

```diff
@@ -1,14 +1,14 @@
 import { inject } from 'vue';
 import { fusionAuthKey } from '#/injectionSymbols';
-import type { FusionAuth } from '#/types';
+import type { FusionAuth, UserInfo } from '#/types';
 
-export const useFusionAuth = (): FusionAuth => {
+export const useFusionAuth = <T extends FusionAuth<UserInfo>>(): FusionAuth => {
```

Comment:
Perhaps try `<T = UserInfo>(): FusionAuth<T>` here. That would give define `T` as the generic, with `UserInfo` being the default value for `T` if none is provided. Then make sure to pass it along to the `FusionAuth` return type. That's what will give it the strong typing!